components/styled-links.js

```diff
@@ -15,9 +15,10 @@ const anchorStyles = {
 }
 
 const activeLinkStyles = (props, theme) => ({
```

Comment:
Oh, actually it _did_ update [the function](https://github.com/kentcdodds/glamorous-website/pull/248/files#diff-30a062bd7a483909fe2b9c221b984af9R46) that calls this one, so the codemod is good 👍 But I updated it all the same :)